Makefile

```diff
@@ -63,6 +63,11 @@ bundle-all-platforms:
 	$(MAKE) GOOS=darwin  GOARCH=amd64 ./bundles/preflight-bundle-darwin-amd64.tgz
 	$(MAKE) GOOS=windows GOARCH=amd64 ./bundles/preflight-bundle-windows-amd64.tgz
 
+# Packages
+
+package-test:
+	opa test -v --explain=notes $(ROOT_DIR)/preflight-packages/examples.jetstack.io
```

Comment:
@j-fuentes, based on the standup comment, there might be some changes to make here?

Reply:
Yes, since now we have #29 merged, packages can be tested with `preflight package test <path to pkg>`.

Reply:
You can use `preflight package test` here if you want, and I'd do that, but in this case, I am not sure if it is better to do:

```
package-test: <whatever target is needed to have preflight binary in place>
  preflight package test $(ROOT_DIR)/preflight-packages/examples.jetstack.io
```

or

```
package-test:
  go run ./main.go package test $(ROOT_DIR)/preflight-packages/examples.jetstack.io
```

BTW, if you proceed to do this, double-check that `preflight package test` supports a directory with packages as argument instead of the path to a package. In case it doesn't support it, I think we should make it support it.

Reply:
I've switched to this in ca3dc849bbe57dae2e2f4d2a295be672f53ae2a4